### DISCUSSION TITLE
android 11 shows a black screen

### DISCUSSION THREAD

**Discussion by fenglilinglegeluan** [osociety/vernet#134] [Q&A]
When I open the app on Android 11 version, it shows a black screen. I tried other version numbers of the app and found that all versions after 1.01 display a black screen. I'm very curious and would like to know why. Looking forward to your reply.

**Comment by git-elliot** [comment]
@fenglilinglegeluan I don't have android 11 as a physical device but I can test vernet on emulator.

**Comment by git-elliot** [comment]
Hi @fenglilinglegeluan, I tested vernet on emulator having android 11 and it is working fine. Can you provide more details about the device you are using.
<img width="368" alt="Screenshot 2024-03-23 at 10 40 28 AM" src="https://github.com/osociety/vernet/assets/18566822/5a6040f2-97a6-4bd9-8808-faf6702b7fd9">

**Comment by fenglilinglegeluan** [comment]
@git-elliot Hi Elliot,
Thank you for your reply. My phone's desktop version is MIUI 12.5, and I suspect that it might be related to this system. Compatibility issues often arise due to the optimization of this system. 
Kind regards,

![Screenshot_2024-03-23-20-32-27-821_com.android.settings.jpg](https://github.com/osociety/vernet/assets/44286100/9aa158de-9686-4c52-a0ab-3713c2f04442)

![Screenshot_2024-03-23-20-19-35-007_com.miui.home.jpg](https://github.com/osociety/vernet/assets/44286100/e3c40c4a-b277-4412-bb29-c33f11c0c8d4)

**Comment by git-elliot** [comment]
Yes, I know about MIUI being so aggressive that it kills the app as soon as it goes in background. Can you turn off battery optimization for vernet and lock this app in recents menu so that OS doesn't kill it. 
Does it happen on launch also or just switching between apps?

**Reply by fenglilinglegeluan** [reply]
Let me try.

**Comment by fenglilinglegeluan** [comment]
@git-elliot I followed all the steps you mentioned, but it didn't work. Maybe it's just an issue with certain phone systems. Let's forget about it.

**Comment by git-elliot** [comment]
@fenglilinglegeluan But I am very disappointed from MIUI and because of that you are not able to use my app. 
I will update android plugins to latest in next update and you can try the next update to see if this issue is fixed or still same.

**Comment by git-elliot** [comment]
https://github.com/osociety/vernet/pull/136

**Comment by git-elliot** [comment]
Splash screen added - https://github.com/osociety/vernet/pull/157
